stix_shifter/requirements.txt

```diff
@@ -1,19 +1,19 @@
 adal==1.2.7
-antlr4-python3-runtime==4.8
-aioboto3==10.3.0
+aioboto3==10.4.0
 aiohttp-retry==2.4.0
 aiomysql==0.0.21
+antlr4-python3-runtime==4.8
 asyncio==3.4.3
 asynctest==0.13.0
+colorlog==6.7.0
 flask==2.2.2
-pyOpenSSL==23.0.0
 flatten_json==0.1.13
+json-fix==0.5.1
+jsonmerge==1.9.0
+pyOpenSSL==23.0.0
 python-dateutil==2.8.2
 requests_toolbelt==0.10.1
+stix2-matcher==3.0.0
 stix2-patterns==1.3.2
-stix2-validator==3.0.2
+stix2-validator==3.1.3
```

Comment:
@delliott90 @mdazam1942 do we have a concern using this latest version of stix2-validator?

Reply:
we never tested actually if existing mappings passes the validation with the new validator. If you can test then i guess we can update this version @baulus

Reply:
test with
```bash
% python main.py execute "stix_bundle" "stix_bundle" '{"type": "identity", "id": "identity--f431f809-377b-45e0-aa1c-6a4751cae5ff", "name": "stix_bundle", "identity_class": "events"}' '{"url": "https://raw.githubusercontent.com/opencybersecurityalliance/stix-shifter/develop/data/cybox/1.json", "options": {"stix_validator": true, "stix_2.1": false}}' '{"auth": {"username": null, "password": null}}' "[ipv4-addr:value = '127.0.0.1']  START t'2019-03-01T10:57:51.826Z' STOP t'2023-03-01T11:02:51.826Z'"
```
with "stix_validator": true/false and "stix_2.1": false/true for 
https://raw.githubusercontent.com/opencybersecurityalliance/stix-shifter/develop/data/cybox/stix-2.1.json
and 
https://raw.githubusercontent.com/opencybersecurityalliance/stix-shifter/develop/data/cybox/1.json